### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by offsset**

### CODE REVIEW

**Comment by offsset** [comment]
https://offsset.github.io/react_decompose/

**Review by tetiana-ivanchenko — CHANGES_REQUESTED** [review]
you didn't add navigation component

**Review by tetiana-ivanchenko — CHANGES_REQUESTED** [review]
still didn't add

image

**Comment by offsset** [comment]
День добрий.Скажить будь ласка, що треба зробить. В мене всi тести проходять.

**Comment by offsset** [comment]
Снимок

File: src/components/Header/Header.jsx [hunk 1]
```diff
@@ -1,6 +1,14 @@
-// import the React library here
-// import a css file containig article styles
+import React from 'react';
+import './Header.css';
+import Navigation from '../Navigation/Navigation';
```
Comment (tetiana-ivanchenko) [inline src/components/Header/Header.jsx:3]: one more time:
2. path is invalid
2. you navigation files didn't added to github

check your tests before requesting review
image

**Review by tetiana-ivanchenko — CHANGES_REQUESTED** [review]

**Pull request closed without merging**